### ISSUE TITLE
Collection name missing in object records

### ISSUE THREAD

**Issue by staplegun** [NationalMuseumAustralia/Collection-API#85]
```javascript
id: "113163",
type: "object",
title: "The Stool Journals",
collection: {
  id: "4193",
  type: "Collection"
},
```

**Comment by Conal-Tuohy** [comment]
The bug is in the XML to RDF mapping: the label is being encoded in RDF as an XML literal!
The SPARQL query
```sparql
describe <http://data.nma.gov.au/object/113163#>
```
returns:
```xml
<rdf:RDF
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:j.0="http://www.cidoc-crm.org/cidoc-crm/"
    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
  <j.0:E19_Physical_Object rdf:about="http://data.nma.gov.au/object/113163#">
    ...
    <j.0:P2_has_type rdf:parseType="Resource">
      <rdfs:label>Journals</rdfs:label>
    </j.0:P2_has_type>
    ...
  </j.0:E19_Physical_Object>
</rdf:RDF>
```

**Comment by Conal-Tuohy** [comment]
Snipped to exclude a lot of other properties, but the "included_in" property is there now:
```json
{
  "@context": "/context.json",
  "id": "http://nma-dev.conaltuohy.com/object/113163#",
  "type": "PhysicalObject",
  "included_in": {
    "id": "http://nma-dev.conaltuohy.com/collection/4193#",
    "type": "http://www.cidoc-crm.org/cidoc-crm/E78_Collection",
    "identified_by": "http://nma-dev.conaltuohy.com/collection/4193#repositorynumber",
    "documented_in": "http://nma-dev.conaltuohy.com/collection/4193",
    "label": "Turner and Valentine Families collection",
    "see_also": "http://nma-dev.conaltuohy.com/object?collection=%22Turner%20and%20Valentine%20Families%20collection%22"
  },
  "label": "The Stool Journals"
}
```
The "simple" JSON is still missing details though:
```json
{
      "collection": {
        "id": "4193",
        "type": "Collection"
      }
}
```

**Comment by Conal-Tuohy** [comment]
ready to deploy to production

**Comment by f27wood** [comment]
I also just updated the above record: 113163 so we have two test records.
Note to self - remember to change them back

**Comment by f27wood** [comment]
Testing this, it looks like the collection info is missing altogether from the simple format. Looks OK for LD format.

**Comment by Conal-Tuohy** [comment]
... which points the finger at the `trix-to-json.xsl` stylesheet.

**Comment by staplegun** [comment]
I think this is our first major victim of following a schema that is still evolving.

In March, the Linked Art website didn't specify how to model objects being inside a curated collection. However there was a Getty Cookbook example using `included_in` (`crm:P106i_forms_part_of`). This is the version we modeled in our mapping `emu-to-crm` and `trix-description-to-dc`. https://linked.art/cookbook/getty/photoarchive/

In April, the Collections page on the Linked Art website was updated to prefer `aggregated_by` (`ore:isAggregatedBy`). However we did not notice this change. https://linked.art/model/collection/

Recently collection titles disappeared (this issue). In an attempt to fix this we mistakenly changed the mapping in `emu-to-crm` to `part_of` (`crm:P46i_forms_part_of`), but not in `trix-description-to-dc`, which is why collection details now appear in the JSON-LD but not at all in the simple format.

We will need to test this a bit further. If we revert to `crm:P106i_forms_part_of` we still need to determine why the titles are missing. If we change to the now preferred `ore:isAggregatedBy`, we will need to check other parts of the mapping are not affected as this predicate is also used elsewhere (we need to double-check the aggregator found is a collection and not something else).
